meshery-design-embed/package.json

```diff
@@ -24,17 +24,17 @@
         "preview": "vite preview"
     },
     "peerDependencies": {
-        "react": "^17.0.2",
-        "react-dom": "^17.0.2"
+        "react": "^18.3.1",
```

Comment:
this will break support for meshery which is on react 17 , replace with ">=17.0.2"

Reply:
@aabidsofi19 We needed React 18 because we need to update layer5.io to React 18 for Gatsby v5
can't we keep the embed version to **0.3.0** for meshery and **0.4.0** for layer5.io

Reply:
Done @aabidsofi19